onnx/optimizer.py

```diff
@@ -10,18 +10,35 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import onnx
 import onnx.onnx_cpp2py_export.optimizer as C  # type: ignore
+from onnx import ModelProto
 
 """Apply the optimization on the serialized ModelProto.
 
 Arguments:
-    input (string): serialized ModelProto
+    input (string or ModelProto): (serialized) model
```

Comment:
The signature should not allow string as input anymore. and the output type should remain as ModelProto

Reply:
good catch